ui/src/components/table/QTable.json

```
@@ -788,7 +788,17 @@
         "cols": {
           "type": "Object",
           "desc": "Column definitions",
-          "__exemption": [ "examples" ]
+          "__exemption": [ "examples" ],
+          "definition": {
+            "name": {
+              "type": "String",
+              "desc": "Column name"
+            },
+            "value": {
+              "type": "String",
+              "desc": "Column value for the given row"
+            }
```

This would limit the object to just `name` and `value`, let's add a "rest" property with an explanation.
```suggestion
            },
            "...props": {
              "type": "Any",
              "desc": "Any other props you defined in 'columns'",
              "__exemption": [ "examples" ]
            }
```